common/dynamicconfig/collection_test.go

```diff
@@ -76,78 +77,114 @@ func (s *collectionSuite) SetupSuite() {
 }
 
 func (s *collectionSuite) TestGetIntProperty() {
```

Comment:
The `property` suffix should be removed from these test names to match the new APIs

Reply:
I was trying to minimize the diff and not change stuff that wasn't really necessary to change for usability.

How about we do another PR to clean up stuff like this after?